src/EVM.hs

```diff
@@ -2589,6 +2565,27 @@ instance VMOps Symbolic where
   toGas _ = ()
   whenSymbolicElse a _ = a
 
+  partial e = assign #result (Just (Unfinished e))
```

Comment:
`partial` now only needs to be defined for symbolic exec

Reply:
In the long run, `partial` should not be part of `VMOps` but it needs a slight refactor to other functions